src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -98,15 +84,30 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
     };
 
     public createMap = () => {
-        const { config } = this.props;
-        const center = get(config, "center", [DEFAULT_LONGITUDE, DEFAULT_LATITUDE] as [number, number]);
+        const {
+            config,
+            geoData: {
+                location: { data },
+            },
+        } = this.props;
+        const center = get(config, "center");
```

Comment:
How about this?
```
const {
            config = {},
            geoData: {
                location: { data },
            },
        } = this.props;
        const { center } = config;
```

Reply:
It said: `Property 'center' does not exist on type '{}'.ts(2339)`.
So I will keep it as is :)

Reply:
So it could be :)
```
const center: IGeoCenter = get<IGeoConfig, "center">(config, "center");
const zoom: number = get<IGeoConfig, "zoom", number>(config, "zoom", DEFAULT_ZOOM);
```

Reply:
updated. thanks